trl/trainer/ddpo_trainer.py

```diff
@@ -0,0 +1,620 @@
+# NOTE: code heavily inspired by https://github.com/kvablack/ddpo-pytorch
```

Comment:
Copy right to add here

Reply:
Yeah should include the license and then include the reference.

Reply:
I would just add something like:

```python
# Copyright 2023 DDPO-pytorch authors, metric-space,  The HuggingFace Team. All rights reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

the DDPO-pytorch repo is under MIT license so should be all good